File: python/season_utils.py
### File: utils/season_utils.py
from datetime import datetime

def get_current_season():
    """Get the current anime season based on the current date."""
    current_date = datetime.now()
    month = current_date.month
    year = current_date.year
    
    if month in [1, 2, 3]:
        return "winter", year
    elif month in [4, 5, 6]:
        return "spring", year
    elif month in [7, 8, 9]:
        return "summer", year
    else:  # month in [10, 11, 12]
        return "fall", year

File: lib/services/season_utils.dart
class SeasonData {
  final String season;
  final int year;
  
  SeasonData({required this.season, required this.year});
}

class SeasonUtils {
  static SeasonData getCurrentSeason() {
    final now = DateTime.now();
    final month = now.month;
    final year = now.year;
    
    String season;
    if (month >= 1 && month <= 3) {
      season = 'winter';
    } else if (month >= 4 && month <= 6) {
      season = 'spring';
    } else if (month >= 7 && month <= 9) {
      season = 'summer';
    } else {
      season = 'fall';
    }
    
    return SeasonData(season: season, year: year);
  }
} 